src/bazel/bazel_task_info.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as vscode from "vscode";
+import { IBazelCommandOptions } from "./bazel_command";
+
+/** Information about a Bazel task. */
+export class BazelTaskInfo {
+  /** pid for the task (if started). */
+  private processId: number;
```

Comment:
Since this and `exitCode` just have trivial accessors/mutators, you can ditch those methods and just make these public fields (and then update the use sites accordingly). We can still safely update their behavior in the future if we need to by declaring them as properties instead of methods.

Reply:
Done